pipelines/validation/h6c/post-lstbin/task_scripts/do_SINGLE_BASELINE_POSTPROCESSING_AND_PSPEC_envmode.sh

```diff
@@ -0,0 +1,99 @@
+#! /bin/bash
```

Comment:
Is there a reason to keep this script around?

Reply:
Nope.